### PR TITLE
[PLA-1558] add change password

### PR DESCRIPTION
**Pull request by zlayine**

### CODE REVIEW

File: resources/js/components/pages/Settings.vue [hunk 1]
```diff
@@ -13,101 +13,16 @@
...
+                        <SettingsApiTokens />
+                        <SettingsWalletAccount />
                     </div>
                 </div>
 
                 <div class="flex flex-col space-y-4 mt-4">
                     <div class="flex justify-between">
                         <h1 class="text-xl">Settings</h1>
                     </div>
+                    <SettingsResetPassword v-if="appStore.isMultiTenant" />
```
Comment (enjinabner) [inline resources/js/components/pages/Settings.vue:25]: I think reset password should be present at all times, right @v16Studios

**Review by enjinabner — CHANGES_REQUESTED** [review]

File: resources/js/components/pages/Settings.vue (hunk 1 above)
Comment (zlayine) [reply resources/js/components/pages/Settings.vue:25]: but the mutation is only available on multi tenant package

Comment (enjinabner) [reply resources/js/components/pages/Settings.vue:25]: Right, we're using token as login for non multi-tenant right?

Comment (zlayine) [reply resources/js/components/pages/Settings.vue:25]: yes

**Pull request merged**